packages/cactus-plugin-ledger-connector-iroha/src/main/json/openapi.json

```diff
@@ -173,9 +173,646 @@
+            "type": "string"
+          },
+          "roleName": {
+            "type": "string"
+          }
+        }
+      },
+      "DetachRoleRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Detach Role.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "roleName"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "roleName": {
+            "type": "string"
+          }
+        }
+      },
+      "GetRolePermissionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Role Permissions.",
+        "type": "object",
+        "required": [
+          "roleId"
+        ],
+        "properties": {
+          "roleId": {
+            "type": "string"
+          }
+        }
+      },
+      "GrantPermissionRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Grant Permission.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "permission"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "permission": {
+            "type": "string"
+          }
+        }
+      },
+      "RevokePermissionRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Revoke Permission.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "permission"
+        ],
+        "properties": {
+          "accountId": {
+            "type": "string"
+          },
+          "permission": {
+            "type": "number"
+          }
+        }
+      },
+      "GetTransactionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Transactions.",
+        "type": "object",
+        "required": [
+          "txHashesList"
+        ],
+        "properties": {
+          "txHashesList": {
+            "type": "array",
+            "items":{
+              "type": "string"
+            }
+          }
+        }
+      },
+      "GetPendingTransactionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Pending Transactions.",
+        "type": "object",
+        "required": [
+          "pageSize"
+        ],
+        "properties": {
+          "pageSize": {
+            "type": "number"
+          },
+          "firstTxHash": {
+            "type": "string"
+          }
+        }
+      },
+      "GetAccountTransactionsRequestParameters": {
+        "description": "The list of arguments to pass in to the transaction request to Get Account Transactions.",
+        "type": "object",
+        "required": [
+          "accountId",
+          "pageSize",
+          "firstTxHash"
```

Comment:
this is optional.

Also there are more optional params:
https://iroha.readthedocs.io/en/develop/develop/api/queries.html#result-pagination
Look to the picture:
![obraz](https://user-images.githubusercontent.com/15332594/184929667-202f552c-f684-4d9b-b9b4-edb5313bd916.png)

Reply:
So shall I add firstTxTime, lastTxTime, firstTxHeight, lastTxHeight as well in the GetAccountTransactionsRequestParameters?

Reply:
> So shall I add firstTxTime, lastTxTime, firstTxHeight, lastTxHeight as well in the GetAccountTransactionsRequestParameters?

I think it would be good to add them as optional params (of course if Cactus can handle that params)